### ISSUE TITLE
Latency of the vector instructions

### ISSUE THREAD

**Issue by soniab** [plctlab/plct-gem5#35]
Hi, 

I would like to know where can I get the information on the Latency of vector instructions in gem5. I am mainly interested in the latency of arithmetic vector instructions and Memory vector instructions used in the Gem5 simulator. 

Regards,
Sonia

**Comment by ksco** [comment]
It's not yet implemented.

**Comment by soniab** [comment]
Ok. Does that mean all the vector instructions are using the constant latency?

**Comment by ksco** [comment]
> Does that mean all the vector instructions are using the constant latency?

Yes. A proper latency model is planned, but we're currently lacking people working on this...
